reference/orders.sf.yml

```diff
@@ -55,12 +55,22 @@ paths:
               type: string
               enum:
                 - lineItems
+                - lineItems.physicalItems
+                - lineItems.physicalItems.socialMedia
+                - lineItems.physicalItems.options
+                - lineItems.physicalItems.categories
```

Comment:
probably out of scope of this PR: should we also update the schema for the "ItemPhysical.categories" : https://github.com/bigcommerce/docs/blob/b874b6059d85db201ded82162e1dd9f6ed641270/reference/orders.sf.yml#L339-L344